model_demos/cv_demos/yolo_v5/pytorch_yolov5_320.py

```diff
@@ -65,11 +67,14 @@ def run_pytorch_yolov5_320(variant="yolov5s"):
     pixel_size = 320  # image pixel size
 
     # Load data sample
-    url = "https://raw.githubusercontent.com/pytorch/hub/master/images/dog.jpg"
-    image = Image.open(requests.get(url, stream=True).raw).convert("RGB")
+    url = "http://images.cocodataset.org/val2017/000000039769.jpg"
+    os.system(f"wget -c -P /tmp/ --read-timeout=5 --tries=0 {shlex.quote(url)}")
```

Comment:
Can wget return the downloaded path? Or we need to manually specify it?

Reply:
Sure will update on upcoming patch